lib/shopify_dashboard_plus.rb

```diff
@@ -49,39 +73,47 @@ def set_connection(key, pwd, name)
       ShopifyAPI::Base.site = shop_url
       shop = ShopifyAPI::Shop.current
       $shop_name = name
-      connection_open
+      open_connection
     rescue
-      connection_closed
+      close_connection
     end
   end
 
-  def connection_closed
+  def close_connection
     $connected = false
   end
 
-  def connection_open
+  def open_connection
     $connected = true
   end
 
-  def connected?
-    $connected
-  end
+  def connected?; $connected; end
 
-  def shop_name
-    $shop_name
-  end
+  def shop_name; $shop_name; end
 
 
   ## Generic Helpers
+  def date_today
+    DateTime.now.strftime('%Y-%m-%d')
+  end
+
+  def days(num)
+    num.to_i == 1 ? "#{num} Day" : "#{num} Days"
+  end
+  
   def hash_to_list(unprocessed_hash)
     return_list = []
-    unprocessed_hash.each do |key, value|
-      return_list.append([key, value])
-    end
+    unprocessed_hash.each {|key, value| return_list.append([key, value])}
   end
 
-  def date_today
-    DateTime.now.strftime('%Y-%m-%d')
+  def strip_protocol(page)
+    page = page.start_with?('http://') ? page[7..-1] : page
+    page = page.start_with?('https://') ? page[8..-1] : page
+  end
+
+  def get_host(site)
+    host = URI(site).host.downcase
+    host = host.start_with?('www.') ? host[4..-1] : host
```

Comment:
```ruby
host.sub(/\Awww\./, "")
```

might also do the trick. 

Also, the second assignment is unnecessary as I'm assuming you only care about the return value.